x-pack/plugins/reporting/export_types/common/layouts/print.css

```diff
@@ -121,8 +124,11 @@ visualize-app .visEditor__canvas {
  * to handle this fine, but firefox moves the visualizations around.
  */
  dashboard-app .react-grid-item {
-  height: 0 !important; /* 1. */
-  width: 0 !important; /* 1. */
```

Comment:
Removing these makes the above block-level comment outdated. 

It looks like it already was outdated though, as there is no server/lib/screenshot file anymore. I am not sure what "can only use the properties that will be manually set..." means